databrowser/src/domain/auth/UserAuthentication.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="isAuthenticated === true">
+    {{ user?.email }}<button @click="onLogout">Logout</button>
+  </div>
+  <div v-else>
+    <button @click="onLogin">Login</button>
+    <button @click="onRegister">Register</button>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed } from 'vue';
+import { useStore } from 'vuex';
+import { keycloak } from './init';
```

Comment:
What do you think about renaming the `init` file to `keycloak`? There we store all the Keycloak logic.